docs/iot/tutorials/gpio-input.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Use GPIO for binary input
+description: Learn how to use GPIO for binary input with the .NET IoT Libraries.
+author: camsoper
+ms.author: casoper
+ms.date: 10/04/2022
+ms.topic: tutorial
+ms.prod: dotnet
+recommendations: false
+---
+
+# Use GPIO for binary input
+
+General-purpose I/O (GPIO) pins can be configured to receive electrical signals as input. At its most basic level, this is useful for scenarios that detect the opening/closing of a circuit. Such circuits might include push buttons, toggle switches, reed switches, pressure switches, and other devices that represent binary (on/off) values by completing a circuit.
+
+In this tutorial, you'll use .NET and your Raspberry Pi's GPIO pins to detect the opening and closing of a circuit.
+
+## Prerequisites
+
+- [!INCLUDE [prereq-rpi](../includes/prereq-rpi.md)]
+- Jumper wires
+- Breadboard (optional)
+- Raspberry Pi GPIO breakout board (optional)
+- [!INCLUDE [tutorial-prereq-dotnet](../includes/tutorial-prereq-dotnet.md)]
+
+[!INCLUDE [ensure-ssh](../includes/ensure-ssh.md)]
+
+## Prepare the hardware
+
+Use the hardware components to build the circuit as depicted in the following diagram:
+
+:::image type="content" source="https://via.placeholder.com/400x200.png" alt-text="A Fritzing diagram showing a circuit that connects a ground pin to pin 21." lightbox="https://via.placeholder.com/800x400.png":::
+
+The image above depicts a direct connection between a ground pin and pin 21.
+
+> [!TIP]
+> The diagram uses a breadboard and GPIO breakout for illustrative purposes, but feel free to just jumper a ground pin and pin 21 on the Raspberry Pi.
+
+[!INCLUDE [tutorial-rpi-gpio](../includes/tutorial-rpi-gpio.md)]
+
+## Create the app
+
+Complete the following steps in your preferred development environment:
+
+1. Create a new .NET Console App using either the [.NET CLI](../../core/tools/dotnet-new.md) or [Visual Studio](../../core/tutorials/with-visual-studio.md). Name it *InputTutorial*.
+
+    ```dotnetcli
+    dotnet new console -o InputTutorial
+    ```
+
+1. [!INCLUDE [tutorial-add-packages](../includes/tutorial-add-packages.md)]
```

Comment:
I'm pretty sure we're missing a step here to `cd` into the newly created _InputTutorial_ project directory. Consider adding this missing step.